### ISSUE TITLE
[Bug] ANDROID Button FontFamily instable and broken (Custom Font?)

### ISSUE THREAD

**Issue by groege** [xamarin/Xamarin.Forms#6419]
### Description
FontFamily on Forms Button is instabile and might be lost at some point (resulting in the text displaying the default font), this happens randomly as you can see in the screenshots.
Also in my project it never works if I have a button within a ListView (it will always display the asian charecters instead of my glyphs)
On iOS this issue does not happen as far as I can tell.

Example Button:
```
<Button HorizontalOptions="End"
              FontFamily="{OnPlatform Android=CarrierMobile.ttf#CarrierMobile, UWP=Assets/Fonts/CarrierMobile.ttf#CarrierMobile, iOS=CarrierMobile}"
              Text="&#xE009;"
              FontSize="Large"
              BackgroundColor="Transparent"
              Clicked="ClickedEditTransportMediums"/>
```

### Expected Behavior
If FontFamily is set it should never revert to the default font.

### Actual Behavior
FontFamily is randomly lost or not working from the beginning (in the case of ListView)

### Basic Information

- Version with issue: Xamarin.Forms 4.0.0.482894
- Last known good version: unknown
- IDE: Visual Studio 2017
- Platform Target Frameworks: 9.0
  - Android: android-ndk-r13b 
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:
### Screenshots

![Screenshot_2019-06-05-14-34-37](https://user-images.githubusercontent.com/15341126/58959611-c1114980-87a5-11e9-8199-b3ddcd5312ff.png)
![Screenshot_2019-06-05-14-49-41](https://user-images.githubusercontent.com/15341126/58959619-c40c3a00-87a5-11e9-86a5-e495f55f9c39.png)

**Comment by samhouts** [comment]
@groege  Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by groege** [comment]
I have copied my projects and removed pretty much everything not needed to test the issue.

[Solution1.zip](https://github.com/xamarin/Xamarin.Forms/files/3260634/Solution1.zip)

When you run the project, press first the plus symbol in the toolbar.
The first time it will work and you will see an edit icon, so go back and press the plus icon once more and this time you should see an Asian character.

Thanks!

**Comment by James3UK** [comment]
Is this the same issue as https://github.com/xamarin/Xamarin.Forms/issues/5543 ?

**Comment by groege** [comment]
> Is this the same issue as #5543 and #6317 ?

Seems like it, I will try the recommended workaround in the following comment tomorrow at work:
https://github.com/xamarin/Xamarin.Forms/issues/6317#issuecomment-498358580

Thanks for the references!!

**Comment by groege** [comment]
https://github.com/xamarin/Xamarin.Forms/issues/6317#issuecomment-498358580

Seems like the workaround works ;)
Thanks you James!

**Comment by groege** [comment]
Closed it since it is related to #6317
